packages/react-native-webrtc/src/withBuildScriptExtMinimumVersion.ts

```diff
@@ -0,0 +1,50 @@
+// Copied from https://github.com/expo/expo-cli/blob/main/packages/config-plugins/src/android/Version.ts
+
+import {
+  ConfigPlugin,
+  WarningAggregator,
+  withProjectBuildGradle,
+} from "@expo/config-plugins";
+
+/** Sets a numeric version for a value in the project.gradle buildscript.ext object to be at least the provided props.minVersion, if the existing value is greater then no change will be made. */
+export const withBuildScriptExtMinimumVersion: ConfigPlugin<{
```

Comment:
I think you can use `import { AndroidConfig } from '@expo/config-plugins'` then `AndroidConfig.Version.withBuildScriptExtMinimumVersion`

Reply:
Refer to this more updated plugin https://github.com/expo/config-plugins/blob/379edcd74cc454faf6ca108ea60faffbf7b3a9c3/packages/react-native-adjust/src/withReactNativeAdjust.ts#L97-L100